client/src/client.rs

```diff
@@ -186,6 +186,13 @@ impl RawTx for String {
     }
 }
 
+/// The different authentication methods for the client.
+#[derive(Clone, Debug, Hash, Eq, PartialEq, Ord, PartialOrd)]
+pub enum MineableTx {
+    RawTx(Transaction),
```

Comment:
Actually this doesn't have to be a `Transaction` but a `RawTx` trait object.

Reply:
This doesn't really work, have to make it a `Box<dyn RawTx>` but still run into errors with sizing

Reply:
Right, it would actually have to be a generic enum.

Reply:
Hmm actually maybe not...